css/components/dashboards/dashboards-gallery.scss

```diff
@@ -0,0 +1,142 @@
+.c-dashboards-gallery-list {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  .thumbnail-list-item {
+    position: relative;
+    height: 200px;
+    border-radius: 4px;
+    background-color: $color-dark-grey;
```

Comment:
Please use `$blue-grey` here instead _(it holds the same value ,but we should take as a reference the CSS vars defined in the file `_settings.scss` and avoid using the ones defined in the file `_deprecated.scss`)_

Reply:
Fixed here: https://github.com/resource-watch/resource-watch/pull/1185/commits/68daf5ece578aa9e34947bc6518d91f784da794a